src/pages/home/home.ts

```diff
@@ -1,14 +1,37 @@
 import { Component } from '@angular/core';
-import { NavController } from 'ionic-angular';
+import { NavController, NavParams, ModalController } from 'ionic-angular';
+import { ArticlesDataProvider } from '../../providers/articles-data/articles-data';
+import { ArticlePage } from '../article/article';
+import 'rxjs/add/operator/map';
+
 
 @Component({
   selector: 'page-home',
-  templateUrl: 'home.html'
+  templateUrl: 'home.html',
+  providers: [ArticlesDataProvider]
 })
 export class HomePage {
+  articles: any = [];
+
+  constructor(
+    private articleData: ArticlesDataProvider,
+    public navCtrl: NavController,
+    public navParams: NavParams,
+    public modalCtrl: ModalController) {
+  }
 
-  constructor(public navCtrl: NavController) {
+  read() {
+    this.modalCtrl.create(ArticlePage).present();
```

Comment:
out of scope for this PR to have the Article show page

Reply:
Also done